test/release/examples/primers/reductions.chpl

```diff
@@ -21,92 +21,120 @@ config const size = 10;    // The size of each side of the array
 
 var A: [1..size, 1..size] real; // The 2D work array
 
-//
-// Fill A with random real values between 0 and 1.
-// Uses the NPB random number generator for historical reasons.
+/*
+   Fill an array with random values
+   --------------------------------
+
+   Fill ``A`` with random real values between 0 and 1.
+   Uses the NPB random number generator for historical reasons.
+*/
+
 fillRandom(A, seed, algorithm=RNG.NPB);
 writeln("A is: "); writeln(A);
 writeln();
 
-//
-// Find the average value of the array, by summing over its elements
-// and dividing by the number of elements it contains.
-//
+/*
+   Find the average value of an array
+   -----------------------------------
+
+   We find the average value of the array by summing over its elements
+   and dividing by the number of elements it contains.
+*/
+
 var eltAvg = (+ reduce A) / size**2;
 writeln("The average element of A has the value ", eltAvg);
 writeln();
 
-//
-// We can find the 1-norm of A by summing over the absolute value of the
-// elements.
-//
-// The expression "abs(A)" creates a new matrix which contains in each of its
-// elements the absolute value of the corresponding element in A.
-// The "+ reduce" clause just sums these up.
-//
+/*
+   Find the 1-norm of an array
+   ---------------------------
+
+   We can find the 1-norm of A by summing over the absolute value of the
+   elements.
+
+   The expression ``abs(A)`` creates a new matrix which contains in each of its
+   elements the absolute value of the corresponding element in A.
+   The ``+ reduce`` clause just sums these up.
+*/
+
 var oneNorm = + reduce abs(A);
 writeln("The 1-norm of A is ", oneNorm);
 writeln();
 
 /*
-The Frobenius norm is the square root of the sum over all elements of their
-respective squares.
+   The Frobenius norm
+   ------------------
+
+   The Frobenius norm is the square root of the sum over all elements of their
+   respective squares.
 
-The expression below can be broken down thus:
+   The expression below can be broken down thus:
 
- 1) The Frobenius norm is the square root of sum quantity.
- 2) The quantity is the sum over all elements of a matrix.
- 3) The matrix is the promotion of A by ``**2``.
-      That is, a matrix each of whose elements the square of the corresponding
-      element in A.
+    1) The Frobenius norm is the square root of some quantity.
+    2) The quantity is the sum over all elements of a matrix.
+    3) That matrix is the promotion of ``A`` by ``**2``.
+	 That is, it's a matrix each of whose elements is the square
```

Comment:
I think this is indented too far, making the line with the `3)` bolded in the html output.